src/main/scala/zio/ftp/Ftp.scala

```diff
@@ -0,0 +1,153 @@
+package zio.ftp
+
+import java.io.{ IOException, InputStream }
+import java.nio.file.attribute.PosixFilePermission
+
+import org.apache.commons.net.ftp.{ FTP, FTPFile, FTPFileFilter, FTPClient => JFTPClient, FTPSClient => JFTPSClient }
+import zio.ftp.settings.FtpSettings
+import zio.stream.{ Stream, ZStream, ZStreamChunk }
+import zio.{ Chunk, Managed, ZIO }
+
+object Ftp {
+
+  trait FtpClient {
```

Comment:
This trait is publicly visible but very effectful. We should wrap its side-effecting methos in `Task` (or more appropriate type).

Reply:
Totally agree, my proposal would be to remove all side effects method and only expose
```scala
 trait FtpClient {
    val instance: JFTPClient
  }
```

like this anyone could extend the library if needed (missing method) and just have to wrap it with `blocking { }`